### DISCUSSION TITLE
how to create an tray application

### DISCUSSION THREAD

**Discussion by nyrf** [slint-ui/slint#933] [Q&A] [answered]
Like electron https://www.electronjs.org/docs/latest/api/tray

**Comment by ogoffart** [comment, marked as answer]
Right now we don't offer such API, but if you're using rust, you can try use one of the other rust crate that does that such as https://lib.rs/crates/tray-item or https://lib.rs/crates/ksni  and hook to show or hide the window when the system tray is pressed.

**Reply by nyrf** [reply]
Thanks.

**Reply by FalkWoldmann** [reply]
Related issue: #6053
